packages/http-apis/src/performance/index.ts

```diff
@@ -223,6 +404,48 @@ export function createPerformanceSwaggerFile(): oas31.OpenAPIObject {
           },
         },
       },
+      "/api/v1/portfolio-summary": {
```

Comment:
I wonder if we should store customer requested endpoints under

`/api/v1/custom/{platformname}/portfolio-summary`

Reply:
So that we explicitly support the idea of `custom` endpoints for platforms. Makes it clear that some endpoints are not part of our core offering. It also stops platforms getting a say over our core api as they can only dictate the results of their custom endpoints